src/main/java/catchMultipleExeptions/Arithmetic.java

```diff
@@ -0,0 +1,20 @@
+package catchMultipleExeptions;
+
+
+public class Arithmetic {
+
+    public static int divide(int numerator, int denominator) throws InvalidInputParamException, EqualNumbersException, ResultZeroException {
+        try {
+            if (denominator == 0)
+                throw new InvalidInputParamException("Denominator can't be zero");
+            else if (denominator > numerator)
+                throw new ResultZeroException("Denominator can't be greater than numerator");
+            else if (denominator == numerator)
+                throw new EqualNumbersException("Denominator can't be equal to numerator");
+            else
+                return numerator/denominator;
+        } catch (Exception e) {
```

Comment:
This is just pointless to throw and exception and try to catch it immediately, especially if no more value added to that.
You can remove `try ... catch` block here.

Reply:
removed